content/source/docs/enterprise/admin/update-tfe-license.html.md

```diff
@@ -0,0 +1,161 @@
+           "ChannelID": "",
+           "Channels": [],
+           "ExpirationTime": "2021-06-30T00:00:00Z",
+           "ExpirationPolicy": "ignore",
+           "IsExpired": false,
+           "IsActivationRequired": false,
+           "ActivationEmail": "",
+           "Fields": [
+               {
+                   "FieldName": "max_hosts",
+                   "FieldTitle": "Maximum Number of Hosts",
+                   "FieldType": "Integer",
+                   "Value": "9999"
+               },
+               {
+                   "FieldName": "min_hosts",
+                   "FieldTitle": "Minimum Number of Hosts",
+                   "FieldType": "Integer",
+                   "Value": "1"
+               }
+           ]
+        }
+        ]
+
+## Update License - Online Installation Type
+
+### Automatic Updates
+
+With the online installation type, Terraform Enterprise can be configured to check for the updated license periodically by using the following steps.
+
+1.  Navigate to the Replicated console, which is located at `https://<<TFE_HOSTNAME>>:8800`
+
+2.  Click on the gear icon on the top-right corner then select the **Console Settings** menu.
+
+    ![Console Settings](./images/console-settings.png)
+
+3.  Scroll down to the **License Sync** section and choose the frequency with which the license should be checked.
+
+    ![License Sync](./images/license-sync.png)
+
+### Manual Updates
+
+#### Replicated Console
+
+1.  Navigate to the Replicated console, which is located at `https://<<TFE_HOSTNAME>>:8800`, click on the gear icon on the top-right corner, and select the **View License** menu.
+
+    ![View License](./images/view-license.png)
+
+2.  Click on **Sync License** to start the operation.
+
+    ![Sync License](./images/sync-license.png)
+
+#### Replicated Command Line Interface
+
+1.  Obtain the newly updated license file from HashiCorp and copy it to the Terraform Enterprise host machine.
+
+2.  Connect to the Terraform Enterprise host machine using SSH.
+
+3.  Use the following command to load the new license.
+
+        $ replicatedctl license-load < /path/to/license.rli
+
+## Update License - Airgap Installation Type
+
+### Replicated Console
+
+1.  Navigate to the Replicated console, which is located at `https://<<TFE_HOSTNAME>>:8800`.
+
+2.  Click on the gear icon on the top-right corner and select the **Console Settings** menu.
+
+    ![Console Settings](./images/console-settings.png)
+
+3.  Select **Airgapped Settings** from the left-hand menu and upload the license.
+
+    ![Upload license](./images/upload-license.png)
+
+### Replicated Command Line Interface
+
+1.  Obtain the updated license and airgap package from HashiCorp and copy them to the Terraform Enterprise host.
+
+2.  Connect to the Terraform Enterprise host machine using SSH.
+
+3.  Execute the following command, updating the paths as needed.
+
+        $ replicatedctl license-load --airgap-package /path/to/bundle.airgap < /path/to/license.rli
```

Comment:
Wrapped in code fence + broke the long line up a bit for readability.

````suggestion
        ```shell
        $ replicatedctl license-load \
            --airgap-package /path/to/bundle.airgap < /path/to/license.rli
        ```
````